### PR TITLE
added done homework_3 file

### PR DESCRIPTION
**Pull request by 213140**
homework 3 was done and uploaded to remote repository. In case of merge with mai nbranch I need your review. Please provide review ASAP. Big thanks

### CODE REVIEW

File: lesson_3/homework_03.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+#Initial string
+#alice_in_wonderland = '"Would you tell me, please, which way I ought to go from here?"\n"That depends a good deal on where you want to get to," said the Cat.\n"I don't much care where ——" said Alice.\n"Then it doesn't matter which way you go," said the Cat.\n"—— so long as I get somewhere," Alice added as an explanation.\n"Oh, you're sure to do that," said the Cat, "if you only walk long enough."'
+
+# task 01 == Розділіть змінну alice_in_wonderland так, щоб вона займала декілька фізичних лінії
+alice_in_wonderland = """'"Would you tell me, please, which way I ought to go from here?"\n"That depends a good deal on where you want to get to," said the Cat.\n"I don't much care where ——" said Alice.\n"Then it doesn't matter which way you go," said the Cat.\n"—— so long as I get somewhere," Alice added as an explanation.\n"Oh, you're sure to do that," said the Cat, "if you only walk long enough."'"""
```
Comment (edanilovets) [inline lesson_3/homework_03.py]: The string should fit into 120 characters on the screen (not only after printing). Use """ or \ to make the string displayed on several lines.

File: lesson_3/homework_03.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+a) 8019 : 8     d) 7248 : 6
+b) 9907 : 9     e) 7128 : 5
+c) 2789 : 5     f) 19224 : 9
+"""
+print(f"rest of 8019 : 8 is: {8019%8}")
+print(f"rest of 9907 : 9 is: {8019%8}")
+print(f"rest of 2789 : 5 is: {8019%8}")
+print(f"rest of 7248 : 6 is: {8019%8}")
+print(f"rest of 7128 : 5 is: {8019%8}")
+print(f"rest of 19224 : 9 is: {8019%8}")
```
Comment (edanilovets) [inline lesson_3/homework_03.py]: You have used the same number in each print

File: lesson_3/homework_03.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+"""
+# task 04
+"""
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?
+"""
+black_sea_area = 436402
+azov_sea_area = 37800
+print(f"Black and Azov seas cover {black_sea_area+azov_sea_area} km2")
```
Comment (edanilovets) [inline lesson_3/homework_03.py]: should be
```
print(f"Black and Azov seas cover {black_sea_area + azov_sea_area} km2")
```

Comment (213140) [reply lesson_3/homework_03.py]: homework_3 file was updated based on feedback

**Pull request merged**